app/views/admin/workshops/_form.html.haml

```diff
@@ -35,6 +35,8 @@
         = f.input :rsvp_open_local_time, as: :string, input_html: { data: { value: @workshop.rsvp_opens_at.try(:time).try(:strftime, '%H:%M') } }
   .row
     .large-6.columns
+      %label Organisers
+      = f.collection_select :organisers, Member.all, :id, :full_name, { selected: @workshop.organisers.map(&:id) }, { multiple: true }
```

Comment:
So, my first PR to the project was accepted in September. I say this because I have no project/business knowledge to fall back on.  

We are making a change in the behaviour of the code. Before `Create` was different from `Update` and with this commit they are the same. It is unusual that we have differences in forms but it's not unheard of. So, have you understood the reasons @ + despo (an original commiter but no longer active) wrote the behaviour the way they did and allowed for that? I say this because the maintainers are going to ask.